### ISSUE TITLE
create a group does not work

### ISSUE THREAD

**Issue by pkchuyen** [philsch/ansible-redshift#6]
Hi,

What i try to do is, create a list of users and group. Then assign these users into the group (with privileges set for this group only)

```- name: Create Group - admin
  redshift_user:
    login_host: "{{ redshift_host }}"
    login_user: "{{ redshift_user }}"
    login_password: "{{ redshift_password }}"
    port: "{{ redshift_port }}"
    db: "{{ redshift_db }}"
    group: admin
```

but i got the error:

```TASK [groups : Create Group - admin] ************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Invalid module input: Internal error: couldn't find user to get id"}
```

So i have to create a user with group on it
```- name: Create User - TestUser
  redshift_user:
    login_host: "{{ redshift_host }}"
    login_user: "{{ redshift_user }}"
    login_password: "{{ redshift_password }}"
    port: "{{ redshift_port }}"
    db: "{{ redshift_db }}"
    user: "testuser"
    password: "passwF0rN3wRsUser"
    group: "admin"
    conn_limit: "10"
    state: present
```

But if I set other user with `state: absent` it will remove `admin` group. 
```- name: Create User - TestUser1
  redshift_user:
    login_host: "{{ redshift_host }}"
    login_user: "{{ redshift_user }}"
    login_password: "{{ redshift_password }}"
    port: "{{ redshift_port }}"
    db: "{{ redshift_db }}"
    user: "testuser1"
    password: "passwF0rN3wRsUser"
    group: "admin"
    conn_limit: "10"
    state: absent
```
It could be a bug i think.

Could you please take a look when have a chance?

and how can I set privs for a user to USAGE and SELECT, INSERT on all tables in a schema?

Thanks

**Comment by philsch** [comment]
Hi @pkchuyen 

First a "rule of thumb": The state `absent` should only be used if you want to *delete* a user/group. As the main feature of this module is to manage users/groups the state impacts this resources to be present/absent. So if you like to change attributes like user-group assignments or privileges, always user the *present* state.

The fact that you can't create a group first and then assign users was indeed a bug.

Applying USAGE on schema and SELECT,INSERT on all tables follows the same schema as described in the documentation (this case has also been added in the Readme now). Just do:
```
privs:
    - schema:USAGE/ALL:SELECT,INSERT
state=present
```
*Note:* As Redshift is designed, this applies only for existing tables in the schema. You have to run your runbook again if a new table is created in the schema!

Bugfix and documentation are available in branch **develop** and will be release with **version 0.2**.

Best,
Philipp

**Comment by philsch** [comment]
Released in **0.2.0**
